### ISSUE TITLE
Plugin website unavailable error

### ISSUE THREAD

**Issue by sfguyfor20** [hjdhjd/homebridge-myq#849]
**Describe The Problem:**
I'm getting a "Plugin website unavailable error".. I've been going for for at least 4 days now... I ended up removing the plugin and then trying to reinstall it... now I can't install the plug in either...

**To Reproduce:**

**Logs:**
9/16/2022, 3:47:55 PMWARNINGplugin site unavailable
9/16/2022, 3:48:02 PMmyQ BridgemyQERRORmyQ API: Unable to access the OAuth authorization endpoint.
9/16/2022, 3:48:02 PMmyQ BridgemyQERRORrequest to https://partner-identity.myq-cloud.com/connect/authorize?client_id=IOS_CGI_MYQ&code_challenge=Lkc1G6OnBh8klVo7kwe5mCv_llLt8TmTKICWoNubQHI&code_challenge_method=S256&redirect_uri=com.myqops%3A%2F%2Fios&response_type=code&scope=MyQ_Residential+offline_access failed, reason: getaddrinfo EAI_AGAIN partner-identity.myq-cloud.com
9/16/2022, 3:48:34 PMWARNINGplugin site unavailable

9/18/2022, 10:15:32 AMWARNINGplugin site unavailable
9/18/2022, 10:18:49 AMWARNINGplugin site unavailable
9/18/2022, 10:18:54 AMWARNINGunhandled rejection: undefined
9/18/2022, 10:18:55 AMWARNINGplugin site unavailable
9/18/2022, 10:18:55 AMWARNINGplugin site unavailable
9/18/2022, 10:20:16 AMWARNINGplugin site unavailable

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

![image](https://user-images.githubusercontent.com/73134472/190917472-93793fa2-dff5-4edd-ba57-94d644322c8a.png)

**Comment by InMBH** [comment]
Is there a fix for this?

**Comment by sfguyfor20** [comment]
It was close automatically..., for some reason... no.. not fixed...

**Comment by hjdhjd** [comment]
As documented many times…myQ API errors happen. These are issues on the myQ side, not on the plugin’s side. They typically resolve themselves over the course of a few days. This plugin is only as good as the API we have to use I’m afraid.

TL;DR: not a bug…it’s an issue on the myQ end, not the homebridge-myq. Cheers.
